maven-surefire-plugin dependency added

astutus opened the pull request:
Closes #1

Commit:
maven-surefire-plugin dependency added

Closes #1

fracz reviewed (changes requested):
I don't like the fact that you have changed indentation from spaces to tabs in the `pom.xml`. [I can hardly see](https://github.com/fracz/enroller-fullstack/pull/2/files) what did you really changed.

Could you reformat it as it was before, please?

astutus commented:
omg, I had reformated using Eclipse's shortcut and I didn't look at "diff" after that. I am sorry, I will correct.

Commit:
pom.xml is reformatted

Instead of 'git revert' the file had been restored using 'git checkout
5495675 ./pom.xml', then maven-surefire-plugin was added once again.

fracz commented:
No student has claimed this error this year, so I suppose it has been fixed... somehow.
